frontend/src/app/testQueue/TestCard/TestCard.test.tsx

```diff
@@ -262,9 +271,42 @@ describe("TestCard", () => {
           },
         ],
       },
+      {
+        internalId: device7Id,
+        name: device7Name,
+        testLength: 15,
+        supportedDiseaseTestPerformed: [
+          ...mockSupportedDiseaseTestPerformedHIV,
+        ],
+        swabTypes: [
+          {
+            name: specimen3Name,
+            internalId: specimen3Id,
+            typeCode: "122555007",
+          },
+        ],
+      },
     ],
   };
 
+  const DEFAULT_DEVICE_ORDER = [
+    device1Name,
+    device2Name,
+    device3Name,
+    device4Name,
+    device5Name,
+    device6Name,
+  ].sort((a, b) => a.localeCompare(b));
+
+  const expectDeviceOrder = (
+    deviceDropdown: HTMLSelectElement,
+    deviceNameOrder: string[] = DEFAULT_DEVICE_ORDER
+  ) => {
+    deviceNameOrder.forEach((deviceName, index) => {
+      expect(deviceDropdown.options[index].label).toEqual(deviceName);
```

Comment:
Many tests were checking the order of the device options and the individual array indices would need to be updated any time there is a new device added to the tests so I just extracted this to its own function